### ISSUE TITLE
bug in Database.clean

### ISSUE THREAD

**Issue by pavlis** [wangyinz/mspass#209]
I found what is pretty clearly a bug in Database.clean.   I ran this:
```
rename={ 'latitude' : 'lat', 'longitude' : 'lon'}
db.clean_collection('source',rename_undefined=rename,verbose=True)
```
When verbose is True, as here, it ran.  However, when I first ran the default (False) clean threw an error.   I have lost the error since it now runs, but I remember it was saying the list "print_messages" was not defined. There appears to be an initialization bug.

I note this could be fixed.  The docker container I'm running this on is a couple weeks old.

**Comment by haruming** [comment]
Good catch Gary. I think the way I was dealing with verbose was somehow inappropriate. I will fix it soon today.

**Comment by wangyinz** [comment]
Closing this as completed.

`Database.clean`/`clean_collection` now initialize optional containers independently of `verbose`, and the database tests explicitly exercise `rename_undefined` with `verbose=False`. The undefined `print_messages` failure described here is covered by the current implementation.
